lib/handlers.js

```diff
@@ -1,6 +1,14 @@
+import stringify from 'json-stringify-safe';
+
+function dataToString(data) {
+    return data && typeof data === 'object'
```

Comment:
Can you please return an empty string if data is undefined (much cleaner that printing undefined)